### DISCUSSION TITLE
Remove GoPro suffix from file names?

### DISCUSSION THREAD

**Discussion by thelifeaquatic** [gopro/labs#352] [General]
Hi,
We can modify file names with the QR code, but is there a way to remove the GoPro suffix? i.e can the file simply be named by the date/time?

Thanks

**Comment by dnewman-gpsw** [comment]
Not available in any released firmware.  The original name was preserved, just in case it could be used (GoPro ecosystem used the names for chaptering.) Although technically just date and time could be used.  The feature has been around for 3+ years, no one has asked for this, what does the extra 8 characters break in your workflow?

**Reply by thelifeaquatic** [reply]
our logger creates a csv file and a .ass file (subtitle) to overlay data on the video file. However, the subtitle file needs to match the video filename exactly to work. We don't know the status of the camera, so can't predict the GoPro suffix. We could probably strip the suffix in post-processing, but would be easier to get it right from the start.
Any assistance much appreciated
Cheers

**Reply by dnewman-gpsw** [reply]
How can you precisely predict date and time, if that is coming form the camera.  If you force an external time and date, then we need the suffice to prevent filename duplication.  Using post file renamer is likely the best path.
